### PR TITLE
Mailjet Subscription custom block and bugfixes.

### PR DESCRIPTION
**Pull request by mitomitov**

### CODE REVIEW

File: includes/mailjet.mail.inc [hunk 1]
```diff
@@ -476,28 +481,27 @@ class MailjetSmtpMailSystem implements MailSystemInterface {
...
     if (!$mailer->Send()) {
       if (variable_get('mailjet_debug')) {
-          watchdog('mailjet',
-                   'Error sending e-mail from @from to @to : @error_message',
-                   array(
-                     '@from' => $from,
-                     '@to' => $to,
-                     '@error_message' => $mailer->ErrorInfo),
-                   WATCHDOG_ERROR);
+        watchdog('mailjet', 'Error sending e-mail from @from to @to : @error_message', array(
```
Comment (FerhanGit) [inline includes/mailjet.mail.inc]: According to PSR-2 - Coding style guide, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

File: mailjet.install [hunk 2]
```diff
@@ -1,83 +1,78 @@
...
+      if (!$phpmailer_path) {
+        $requirements['phpmailer'] = $error_data;
+      }
+    }
+    elseif (!file_exists('sites/all/libraries/phpmailer/PHPMailerAutoload.php')) {
+      $requirements['phpmailer'] = $error_data;
+    }
+  }
+  return $requirements;
+}
```
Comment (FerhanGit) [inline mailjet.install]: All text files should end in a single newline (\n). This should be applied to all modified files.

File: README.md [hunk 3]
```diff
@@ -1,202 +0,0 @@
-drupal-mailjet-module-apiv3
```
Comment (gisharata) [inline README.md]: Why this file has been removed ?

File: README.txt [hunk 4]
```diff
@@ -1,39 +0,0 @@
----
```
Comment (gisharata) [inline README.txt]: Why this file has been removed ?

File: includes/mailjet.iframes.inc [hunk 5]
```diff
@@ -1,19 +1,29 @@
...
+ /**
+ * @file
+ * Helper functiions for Registration Iframe.
+ */
 
 /**
- * Registration iframe form.
- */
+ * Regisration iframe block code.
+  */
```
Comment (gisharata) [inline includes/mailjet.iframes.inc]: There is an extra space

File: includes/mailjet.mail.inc [hunk 6]
```diff
@@ -589,16 +596,16 @@ class MailjetSmtpMailSystem implements MailSystemInterface {
...
+    $string_length = drupal_strlen($substring) - 1;
 
     if ($substring[$string_length] == $ending_character) {
       $substring = drupal_substr($substring, 0, $string_length);
     }
 
     return $substring;
   }
-}
+
```
Comment (gisharata) [inline includes/mailjet.mail.inc]: Always put empty line in the end of the file.

File: mailjet.install [hunk 7]
```diff
@@ -1,83 +1,78 @@
...
-        $requirements['phpmailer'] = $errorData;
-      }
-    }
-    elseif (!file_exists('sites/all/libraries/phpmailer/PHPMailerAutoload.php')) {
-      $requirements['phpmailer'] = $errorData;
-    }
-  }
-  return $requirements;
-}
+<?php
```
Comment (gisharata) [inline mailjet.install]: EOL conversion should be Unix(LF) instead of Windows(CR RF)

File: modules/event/mailjet_event.install [hunk 8]
```diff
@@ -1,45 +1,49 @@
...
-        'table' => 'users',
-        'columns' => array('uid' => 'uid'),
-      ),
-    ),
-  );
-
-  return $schema;
-}
\ No newline at end of file
+<?php
```
Comment (gisharata) [inline modules/event/mailjet_event.install]: EOL conversion should be Unix(LF) instead of Windows(CR RF)

File: modules/list/mailjet_list.module [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
-  );
-
-  return $items;
-}
-
-/**
- * Adds permissions for the lists.
- */
-function mailjet_list_access(){}
+<?php
```
Comment (gisharata) [inline modules/list/mailjet_list.module]: EOL conversion should be Unix(LF) instead of Windows(CR RF)

File: includes/mailjet.iframes.inc [hunk 10]
```diff
@@ -27,16 +37,16 @@ function mailjet_plans_iframe() {
...
-      
+
     global $language;
-  
-    $langCodesMap = array('en' => 'en_US', 'fr' => 'fr_FR', 'de' => 'de_DE', 'es' => 'es_ES');
-    $defaultLang = 'en';
-    $locale = isset($langCodesMap[$language->language]) ? $langCodesMap[$language->language] : $langCodesMap[$defaultLang];
 
-    $callbackurl = 'r=commerceguys&t='.$token.'&show_menu=none&sp=display&locale='.$locale;
+    $lang_codes_map = array('en' => 'en_US', 'fr' => 'fr_FR', 'de' => 'de_DE', 'es' => 'es_ES');
```
Comment (gisharata) [inline includes/mailjet.iframes.inc]: Each array variable must be in a new row

**Pull request merged**